emulator/blockchain.go

```diff
@@ -711,6 +725,7 @@ func bootstrapLedger(
 	ctx = fvm.NewContextFromParent(
 		ctx,
 		fvm.WithAccountStorageLimit(false),
+		fvm.WithEVMEnabled(false),
```

Comment:
Should this use the configuration option?

Reply:
For this one it should always be false to over come the chicken egg problem. I'll add some docs to it

Reply:
added some details.